root/utility/age.js

```diff
@@ -28,7 +28,7 @@ export function hasAge<+T: $ReadOnly<{...DatePeriodRoleT, ...}>>(entity: T): boo
    * If there is no begin year, there is no age.
    * Only compute ages when the begin date is AD.
```

Comment:
```suggestion
```

Also we have some tests for `age` in root/static/scripts/tests/utility.js, would be nice to add some that test negative years